packages/core/contracts/financial-templates/perpetual-multiparty/ConfigStore.sol

```diff
@@ -160,17 +168,24 @@ contract ConfigStore is ConfigStoreInterface, Testable, Lockable, Ownable {
         // before a vulnerability drains its collateral.
         require(config.timelockLiveness <= 7 days && config.timelockLiveness >= 1 days, "Invalid timelockLiveness");
 
-        // Upper limits for the reward and bond rates are estimated based on offline discussions,
-        // and it is expected that these hard-coded limits can change in future deployments.
+        // Upper limits for the reward are estimated based on offline discussions,
+        // and it is expected that these hard-coded limits change in future deployments.
         // For a discussion thread, go [here](https://github.com/UMAprotocol/protocol/pull/2223#discussion_r530692149).
 
-        // Proposer bond of 0.04% is based on a maximum expected funding rate error of 200%/year.
-        FixedPoint.Unsigned memory maxProposerBond = FixedPoint.fromUnscaledUint(4).div(1e4);
-        require(config.proposerBondPct.isLessThan(maxProposerBond), "Invalid proposerBondPct");
-
         // Reward rate should be less than 100% a year => 100% / 360 days / 24 hours / 60 mins / 60 secs
         // = 0.0000033
         FixedPoint.Unsigned memory maxRewardRatePerSecond = FixedPoint.fromUnscaledUint(33).div(1e7);
         require(config.rewardRatePerSecond.isLessThan(maxRewardRatePerSecond), "Invalid rewardRatePerSecond");
+
+        // We don't set a limit on the proposer bond because it is a defense against dishonest proposers. If a proposer
+        // were to successfully propose a very high or low funding rate, then their PfC would be very high. The proposer
+        // could theoretically keep their "evil" funding rate alive for 74 hours by continuously disputing honest
```

Comment:
nit: a proposer can theoretically keep disputing indefinitely and a new funding rate will never come in. There is some logic we can add in so that the DVM settlement actually pushes the new funding rate into the contract but it was stripped for bytecode.

Reply:
Each dispute can effectively delay a funding rate for 2 hours.

Reply:
Oh shoot I thought the DVM settlement pushed the funding rate into the contract? That was a core assumption in calculating our PfC, any reason not to add this?

Reply:
Added this comment, we resolved this offline FYI. Our PfC calculation found [here](https://github.com/UMAprotocol/protocol/issues/2039#issuecomment-719734383) actually calculates the proposal bond per proposal liveness window (i.e. 2 hours)

Reply:
+1 -- thanks for checking up on this.